tests/lti/test_routes.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from lti.routes import includeme
+
+import mock
+
+
+class TestIncludeMe(object):
+
+    def test_it_adds_some_routes(self):
+        config = mock.MagicMock()
+
+        includeme(config)
+
+        expected_calls = [
+            mock.call.add_route('about', '/'),
+            mock.call.add_route('token_callback', '/token_callback'),
+            mock.call.add_route('refresh_callback', '/refresh_callback'),
+            mock.call.add_route('config_xml', '/config.xml'),
+            mock.call.add_route('lti_credentials', '/lti_credentials'),
+            mock.call.add_route('lti_setup', '/lti_setup'),
+            mock.call.add_route('lti_submit', '/lti_submit'),
+            mock.call.add_route('lti_export', '/lti_export'),
+            mock.call.add_route('lti_serve_pdf', '/viewer/web/{file}.pdf'),
+            mock.call.add_route('catchall_pdf', '/viewer/*subpath'),
+        ]
+        for call in expected_calls:
+            assert call in config.mock_calls
```

Comment:
This test doesn't really add much but it just keeps our code coverage % score up by making sure the `routes.py` file is run by the tests, means that the tests will fail if `routes.py` is crashing, and keeps codecov happy. The way the test is written, if we add any new routes we won't need to add them to the test. If any of these existing routes are deleted or changed the test will need to be updated.